docs/UserGuide.adoc

```diff
@@ -53,7 +53,81 @@ e.g. typing *`help`* and pressing kbd:[Enter] will open the help window.
 
 === Recipe inventory commands
 
-=== Database commands
+=== Cookbook commands
+==== View a recipe from the cookbook
+View a recipe, its ingredients, and the steps to cook it.
+
+- Format: `cookbook view recipe INDEX`
+- Example:
+* `cookbook view recipe 1`
+
+==== Tag a recipe
+Tag a recipe.
+
+- Format: `cookbook tag recipe INDEX t/TAG…`
+- Examples:
+* `cookbook tag recipe 1 t/Carbonara`
+* `cookbook tag recipe 1 t/Carbonara t/Pasta t/Cream`
+
+==== Untag a recipe
+Untag a recipe.
+
+[TIP]
+If no tags are specified, all the tags for the recipe will be removed.
+
+- Format: `cookbook untag recipe INDEX [t/TAG]…`
+- Examples:
+* `recipe untag 1`
+* `recipe untag 1 t/Carbonara`
+
+==== Search recipes by keyword
+Search for a recipe by a keyword, and the desired recipe can be viewed using the command `cookbook view recipe`.
+
+- Format: `cookbook search recipe w/KEYWORD`
+- Example:
+* `cookbook search recipe w/Carbonara`
+
+==== Search recipes by tag
+Search for recipes by tags, and the desired recipe can be viewed using the command `cookbook view recipe`.
+
+- Format: cookbook search tag [t/TAG]…
+- Example:
+* `cookbook search tag t/Pasta t/Cream t/Easy`
+
+==== Search recipes by ingredients owned
+Search for recipes by the percentage of required ingredients owned, and the desired recipe can be viewed using the command `cookbook view recipe`.
+
+- Format: `cookbook search inventory`
+- Example:
+* `cookbook search inventory`
+
+==== Cook recipe according to a serving size
+Cook and view a recipe from the database according to a serving size.  Once this command is executed, the ingredients used for the recipe will be deducted from the ingredients inventory.
+
+[TIP]
+If no serving size is provided, the default serving size in the recipe will be used.
+- Format: `cookbook recipe INDEX serving [v/SERVING_SIZE]` +
+- Examples: +
+* `cookbook recipe 1 serving`
+* `cookbook recipe 1 serving v/5`
+
+==== Add missing ingredients in a recipe to a shopping list
+Add the missing ingredients in a recipe to a shopping list according to a serving size.
+
+[TIP]
+If no serving size is provided, the default serving size in the recipe will be used.
+
+- Format: `cookbook recipe INDEX shop [serving v/SERVING_SIZE]`
+- Examples:
+* `cookbook recipe INDEX shop`
+* `cookbook recipe INDEX shop serving v/5`
+
+==== Store nutritional information of a recipe for a day
+Tracks the nutritional information of an existing recipe for a specific date
```

Comment:
maybe add a fullstop to be consistent with other commands

Reply:
fixed